src/malware_prediction/main.py

```diff
@@ -41,6 +42,11 @@ def run():
     trainer.train(20, X_train, y_train, verbose=True)
```

Comment:
Hey @1p4pk , hier schmeißt der er eine Exception wegen `X_train`, wenn ich `mpirun -n 2 python3 main.py run_model` starte.

Reply:
Ok @raminqaf , hab ich gesehen. Ich hab dein Zeugs mal bei mir in den Branch gemerged. Also könnten wir dann den einfach in Master mergen. Kannst dir ja mal den feature/distributed-frequency-encoding branch anschauen!

Reply:
Danke dir @1p4pk. Dann lösche ich diesen Branch, wenn du alles darüber auf dem anderen Branch hast. Ok?

Reply:
+1